src/Api/Controllers/LicensesController.cs

```diff
@@ -53,22 +58,30 @@ public async Task<UserLicense> GetUser(string id, [FromQuery] string key)
         return license;
     }
 
+    /// <summary>
+    /// Used by self-hosted installations to get an updated license file
+    /// </summary>
     [HttpGet("organization/{id}")]
-    public async Task<OrganizationLicense> GetOrganization(string id, [FromQuery] string key)
+    public async Task<OrganizationLicense> GetOrganization(string id, [FromBody] SelfHostedOrganizationLicenseRequestModel model)
```

Comment:
suggestion: if this is specifically for Self Hosted we could name it something more appropriate. We went with just `Sync` for the sponsorships, maybe something similar?

Reply:
This endpoint is on cloud, the request is coming from the selfhosted instance.

Re: the `Sync` wording specifically, it's a bit different to sponsorships because sponsorships is pushing data, this is just getting it (which is why I maintained the "get" wording in this method). I guess the idea of a "sync" can apply to both situations though.

I'll rename it `OrganizationSync`.